acme/acme/client.py

```diff
@@ -874,6 +874,12 @@ def revoke(self, cert, rsn):
         """
         return self.client.revoke(cert, rsn)
 
+    def external_account_required(self):
```

Comment:
EAB is acmev2-only, right? If so, this should really be on `ClientV2`.

Reply:
The tests should still work.

Reply:
Yep, the EAB feature is V2 only.
But I was thinking that this code line[0] will be executed regardless of, if the server is a ACMEv1 or ACMEv2 server. 
Am I misunderstanding something?

[0]. https://github.com/robaman/certbot/blob/8adf23283c77dca2b3ecb7b2b5f9d0cc58c6a57d/certbot/client.py#L217

Reply:
ah, good point. so the way the code's set up here is that the `Backwards...` class just translates between v1 and v2 styles, and calls into the appropriate method. I know that seems like a lot of structure for what's only a few lines, but the idea is that some day v1 will be deprecated and `ClientV2` will be a perfect replacement for `Backwards...`. Also, the ACME library is used on its own by external clients, and we want those clients to be able to use the `ClientV2` class directly.

Reply:
Working on moving this into ClientV2.

Reply:
You're totally right, that was a previous version that just automatically passed things through. Now, it's probably best to make it so it's both in ClientV2 and explicitly passed through in BackwardsCompatible, with just returning False if we're acmev1 to avoid complications.

Reply:
Fixed now.